### PR TITLE
Set the default_value of JAVA_HOME for running RPS

### PR DESCRIPTION
**Pull request by ljainpivotalio**
Third try ;-(

### CODE REVIEW

File: ranger-plugin/scripts/enable-ranger-plugin.sh [hunk 1]
```diff
@@ -211,6 +211,28 @@ function update_ranger_url() {
...
+    if [[ ${DIR_NAME} ]]; then
+      JAVA_HOME_DIR="${jdk64}/${DIR_NAME}"
+    fi
+  elif [[ -d ${java_sdk} ]]; then
+    JAVA_HOME_DIR="${java_sdk}"
+  fi
+
+  if [[ ${JAVA_HOME_DIR} ]]; then
+    local prop_file=$(dirname ${SCRIPT_DIR})/etc/rps.properties
+    sed -i -e "s|/usr/java/default|${JAVA_HOME_DIR}|g" ${prop_file}
```
Comment (interma) [inline ranger-plugin/scripts/enable-ranger-plugin.sh]: Seems no `/usr/java/default` existed in rps.properties?
I refer here:https://github.com/apache/incubator-hawq/blob/master/ranger-plugin/conf/rps.properties

Is there something wrong?

Comment (ljainpivotalio) [inline ranger-plugin/scripts/enable-ranger-plugin.sh]: Good catch. Forgot to commit the other 2 files :-(

**Comment by edespino** [comment]
+1 Thanks for working on this quickly.

**Comment by interma** [comment]
LGTM now, +1

Thanks lav!

**Pull request closed without merging**